Community-Aktivitäten/README.md

```diff
@@ -165,6 +165,46 @@ Dauer: 2 Stunden (bei Bedarf auch gerne mehr)
 
 Ausstattung: großer Screen/Beamer, Internet
 
+## Building your career: Ask anything
+:us: :de:
+
+### Collaborators
+* [JUG Schweiz]: [Ixchel Ruiz]
+* [JUG Dortmund]: [Sandra Parsick]
+* [JUG Dortmund]: [Hendrik Ebbers]
+
+### Description:
+If you're looking for career advice in software development, a good strategy is to ask your seniors. They have valuable experience and can provide insights that will help guide your professional development. Their first-hand knowledge can give you a clear understanding of the industry and its requirements. Don't hesitate to join this event with any questions or concerns you may have. By seeking the guidance and advice of our guests, you'll be better equipped to make informed decisions about your career path.
+
+### Duration : 2 hr : 1 hr Tuesday & Wednesday during the conference ( 2 Days ) 
+
+### Beschreibung:
+If you're looking for career advice in software development, a good strategy is to ask your seniors. They have valuable experience and can provide insights that will help guide your professional development. Their first-hand knowledge can give you a clear understanding of the industry and its requirements. Don't hesitate to come to this event with any questions or concerns that you may have. By seeking the guidance and advice of our guests, you'll be better equipped to make informed decisions about your career path.
```

Comment:
```suggestion
Du suchst nach Karrieretipps für deine nächsten Schritte als Softwareentwickler? Manchmal ist es eine gute Strategie, hier die “alten Hasen” zu fragen. Sie verfügen über wertvolle Erfahrungen und können dir Einblicke geben, die für eine berufliche Entwicklung hilfreich sein können. Ihr Wissen aus erster Hand kann dir ein besseres Verständnis der Branche und ihrer Anforderungen vermitteln. Zögere daher nicht, zu dieser Veranstaltung zu kommen, wenn du Fragen oder Bedenken bzgl. deiner Karriere hast. Unser Ziel ist es, dass du nach diesem Meeting besser gerüstet bist, um fundierte Entscheidungen über deine berufliche Laufbahn zu treffen.
```

Reply:
I do really want to keep both descriptions as the event will be held in the two languages!

Reply:
Understood. Unfortunately I cannot guarantee the the pipeline processing the content will actively pick the right one. It could happen that they pick the German one and auto-translate it, still. Let's see what happens. 😅